src/contracts/Input/Value/InputPayload.php

```diff
@@ -16,9 +16,9 @@ final class InputPayload implements \Stringable
 
     private string $content;
 
-    private ?string $name;
+    private string $name;
 
-    public function __construct(string $mediaType, string $format, string $content, ?string $name = null)
+    public function __construct(string $mediaType, string $format, string $content, string $name)
```

Comment:
Doesn't this break current code? Do we always use `loadPayload` to create new instance?

Reply:
![image](https://github.com/ibexa/test-rest/assets/3183926/3a95f149-472d-4ca7-9c6e-7caf73be6662)

Checked by searching for `new InputPayload` in 4.6.x-dev project directory.

Reply:
Reverted those changes to reduce chances of breakage.